charts/graylog/values.yaml

```diff
@@ -413,11 +413,6 @@ graylog:
   ##
   rootUsername: "admin"
 
-  ## Graylog root password
-  ## Defaults to a random 16-character alphanumeric string if not set
-  ##
-  # rootPassword: ""
```

Comment:
Why Graylog root password is remove?

Reply:
my bad! removed not intentionally, it's back!